### PR TITLE
New profile for Creality Ender 3 V3 SE

### PR DESCRIPTION
**Pull request by bistory**
Create new profile for Creality Ender 3 V3.
This profile has been tested, it is based on Creality Print profile.
Orca provides faster and cleaner prints!

### CODE REVIEW

File: resources/profiles/Creality.json [hunk 1]
```diff
@@ -410,22 +278,6 @@
 			"name": "0.36mm SuperChunky @Creality CR-6 0.6",
 			"sub_path": "process/0.36mm SuperChunky @Creality CR-6 0.6.json"
 		},
-		{
-			"name": "0.36mm Chunky @Creality Ender5Pro (2019) 0.5",
```
Comment (SoftFever) [inline resources/profiles/Creality.json:414]: @bistory 
I noticed you have deleted all Creality Ender5Pro (2019) profiles, is this intentional?

Comment (bistory) [reply resources/profiles/Creality.json:414]: Not at all 😅

Comment (bistory) [reply resources/profiles/Creality.json:414]: I think it's due to a bad merge, I will fix that ;-)

**Comment by bistory** [comment]
Please don't merge it yet, I still have to find an elegant way to implement specific volumetric speed for each model.
Bambulab did it by multiplying files for each profile/model, maybe we should follow that path ?
So create a base filament profil, inherit it for each model and tune volumetric speed into those files ?

**Comment by SoftFever** [comment]
> Please don't merge it yet, I still have to find an elegant way to implement specific volumetric speed for each model. Bambulab did it by multiplying files for each profile/model, maybe we should follow that path ? So create a base filament profil, inherit it for each model and tune volumetric speed into those files ?

Yeah, the only way is to create a model specific derivative like this one, but practically we would just create two varieties: normal and HF though...
https://github.com/SoftFever/OrcaSlicer/blob/main/resources/profiles/Prusa/filament/Prusa%20Generic%20PLA%20%40MK4.json

**Comment by bistory** [comment]
> > Please don't merge it yet, I still have to find an elegant way to implement specific volumetric speed for each model. Bambulab did it by multiplying files for each profile/model, maybe we should follow that path ? So create a base filament profil, inherit it for each model and tune volumetric speed into those files ?
> 
> Yeah, the only way is to create a model specific derivative like this one, but practically we would just create two varieties: normal and HF though... https://github.com/SoftFever/OrcaSlicer/blob/main/resources/profiles/Prusa/filament/Prusa%20Generic%20PLA%20%40MK4.json

Thanks for your reply, I followed that way, it's the cleanest way.
Creality has 3 Ender 3 V3 models : the first one is not using the same hotend as the other ones so in theory, the volumetric flow is different. However, due to its limitation, it is virtually impossible to reach the maximum flow, so I considered the 3 models have the same flow (I will make a PR for each model).
I will soon also commit some cleanups regarding K1/K1 Max profiles, because they also lack some settings.

This way, if the user select "Creality Generic PLA", he will have the correct settings (temp, flow, min layer time, ...) for his printer and not generic settings for Ender 3 v1...

I can now consider this work done !

File: resources/profiles/Creality/filament/Creality Generic ABS.json [hunk 2]
```diff
@@ -25,14 +25,7 @@
 		"Creality Ender-3 S1 Pro 0.4 nozzle",
 		"Creality Ender-5 0.4 nozzle",
 		"Creality Ender-5 Plus 0.4 nozzle",
-		"Creality Ender-5 Pro (2019) 0.2 nozzle",
```
Comment (SoftFever) [inline resources/profiles/Creality/filament/Creality Generic ABS.json:28]: @bistory 
Looks like there are still some merge issues?

File: resources/profiles/Creality/filament/Creality Generic PETG.json [hunk 3]
```diff
@@ -35,14 +35,7 @@
 		"Creality Ender-3 S1 Pro 0.4 nozzle",
 		"Creality Ender-5 0.4 nozzle",
 		"Creality Ender-5 Plus 0.4 nozzle",
-		"Creality Ender-5 Pro (2019) 0.2 nozzle",
```
Comment (SoftFever) [inline resources/profiles/Creality/filament/Creality Generic PETG.json:38]: here

File: resources/profiles/Creality/filament/Creality Generic PLA.json [hunk 4]
```diff
@@ -26,14 +26,7 @@
 		"Creality Ender-3 S1 Pro 0.4 nozzle",
 		"Creality Ender-5 0.4 nozzle",
 		"Creality Ender-5 Plus 0.4 nozzle",
-		"Creality Ender-5 Pro (2019) 0.2 nozzle",
-		"Creality Ender-5 Pro (2019) 0.25 nozzle",
-		"Creality Ender-5 Pro (2019) 0.3 nozzle",
 		"Creality Ender-5 Pro (2019) 0.4 nozzle",
-		"Creality Ender-5 Pro (2019) 0.5 nozzle",
-		"Creality Ender-5 Pro (2019) 0.6 nozzle",
```
Comment (SoftFever) [inline resources/profiles/Creality/filament/Creality Generic PLA.json:34]: here

**Comment by bistory** [comment]
It should be good now ;-)

**Review by SoftFever — APPROVED** [review]
Looks good.
Thanks!

**Pull request merged**